_rest_routes.inc.php

```diff
@@ -6222,15 +6224,29 @@
      *          response="401",
      *          ref="#/components/responses/unauthorized"
      *      ),
-     *      security={{"openemr_auth":{}}}
+     *      security={{"openemr_auth":{"user/employer.read", "patient/employer.read"}}}
      *  )
      */
     "GET /api/patient/:puuid/employer" => function ($puuid, HttpRestRequest $request) {
+        if (!UuidRegistry::isValidStringUUID($puuid)) {
```

Comment:
Only thing I could think of would be an architectural improvement to build a Validator class for GET requests.  We use the Validator on insert/update requests (@see PatientValidator).  FHIR will catch if the uuid is invalid, but these standard apis probably throw an exception if the UUID is invalid which is what you're checking here so I think this is fine.

Reply:
This basic pattern would be great to apply for all GETs, though I think that it may be tricky to do in a general way given the variance in possible inputs. Some are UUID, some are other string IDs, etc. Having some common methods, perhaps using an Attribute pattern could be useful.